program-workflows/hiv-recency-testing.md

```diff
@@ -64,9 +64,98 @@ The reporting tools include the national HMIS tools are:
 * HTS Client Card Export - CSV file containing all the fields entered on the HTS Card 
 * Recency Data Export - CSV file with recency data fields 
 
+
+
 ## Data Sharing with Central Server
 
-This is an automated process that occurs every hour until a successful submission is done in a day. Due to the small numbers involved, all the HTS records are sent each time, a process which will be improved as the program proceeds
+This is an automated process that occurs every hour until a successful submission is done in a day. Due to the small numbers involved, all the HTS records are sent each time, a process which will be improved as the program proceeds. The EMR gives the option to change the scheduled sync frequency.
+
+#### Sync data flow
```

Comment:
@mugumedavid This needs to be 3 hashes for Heading 3

Reply:
Actually, the main heading that takes 3 hashes is the one I found "Data Sharing with Central Server" All headings I introduced are under that heading.

Reply:
Oh okay. I skipped from 2 to 4 hashes. Got it.